### PR TITLE
Cypress tests: Runs - Restore & Archive

### PR DESCRIPTION
**Pull request by jpuzz0**
Closes: [RHOAIENG-3886](https://issues.redhat.com/browse/RHOAIENG-3886)
Starts: [RHOAIENG-3887](https://issues.redhat.com/browse/RHOAIENG-3887)

## Description
Test scenarios covered from Active & Archived run list pages:
- single Restore
- multiple Restore
- single Archive
- multiple Archive

Also includes:
- Fix for `explore.cy.ts` test that is unrelated and not owned by our scrum team, but is impeding on CI builds.
- A simple name change of `Create Schedule` -> `Schedule run`, which is a part of [RHOAIENG-3887](https://issues.redhat.com//browse/RHOAIENG-3887), but only captures a small amount that still could change more as a part of that ticket which is not yet fully vetted.

## How Has This Been Tested?
Cypress tests, regression tests

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has added tests or explained why testing cannot be added (unit tests & storybook for related changes)

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

File: frontend/src/__tests__/cypress/cypress/e2e/pipelines/PipelineRuns.cy.ts [hunk 1]
```diff
@@ -111,251 +118,372 @@ const mockJobs = [
...
+        activeRunsTable.selectRowActionByName(runToArchive.display_name, 'Archive');
 
-        // Select the "Experiment" filter, enter a value to filter by
-        pipelineRunsGlobal
-          .findActiveRunsToolbar()
-          .within(() => pipelineRunsGlobal.selectFilterByName('Experiment'));
-        pipelineRunsGlobal
-          .findActiveRunsToolbar()
-          .within(() => pipelineRunFilterBar.findExperimentInput().type('Experiment 1'));
+        activeRunsTable.mockGetRuns([mockActiveRuns[1]], [runToArchive]).as('refreshAllRuns');
```
Comment (Gkrumbach07) [inline frontend/src/__tests__/cypress/cypress/e2e/pipelines/PipelineRuns.cy.ts:174]: why do you save `refreshAllRuns` but dont wait for it anywhere

Comment (Gkrumbach07) [reply frontend/src/__tests__/cypress/cypress/e2e/pipelines/PipelineRuns.cy.ts:174]: i would expect if you are saving it like this that you would wait after the submit to see that the request body is accurate to what was selected

File: frontend/src/__tests__/cypress/cypress/pages/pipelines/archiveRunModal.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    super('Archive run?');
+    this.testId = testId;
+  }
+
+  find() {
+    return cy.findByTestId(this.testId).parents('div[role="dialog"]');
+  }
+
+  findConfirmInput() {
+    return cy.findByTestId('confirm-archive-input');
```
Comment (Gkrumbach07) [inline frontend/src/__tests__/cypress/cypress/pages/pipelines/archiveRunModal.ts:16]:
```suggestion
    return this.find().findByTestId('confirm-archive-input');
```

should it be finding within the modal?

File: frontend/src/__tests__/cypress/cypress/pages/pipelines/restoreRunModal.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+class RestoreRunModal extends Modal {
+  protected testId;
+
+  constructor(testId = 'restore-run-modal') {
+    super('Restore run?');
+    this.testId = testId;
+  }
+
+  find() {
+    return cy.findByTestId(this.testId).parents('div[role="dialog"]');
```
Comment (Gkrumbach07) [inline frontend/src/__tests__/cypress/cypress/pages/pipelines/restoreRunModal.ts:12]:
```suggestion
    return this.find().findByTestId(this.testId).parents('div[role="dialog"]');
```
same suggestion as above

**Review by Gkrumbach07 — COMMENTED** [review]
just a few comments but otherwise looking good

File: frontend/src/__tests__/cypress/cypress/pages/pipelines/archiveRunModal.ts (hunk 2 above)
Comment (jpuzz0) [reply frontend/src/__tests__/cypress/cypress/pages/pipelines/archiveRunModal.ts:16]: Yes, it should, I'll update.

File: frontend/src/__tests__/cypress/cypress/pages/pipelines/restoreRunModal.ts (hunk 3 above)
Comment (jpuzz0) [reply frontend/src/__tests__/cypress/cypress/pages/pipelines/restoreRunModal.ts:12]: This function is meant to replace the "find" of Modal for this instance/sub-class of Modal, so I don't think it makes sense here.

**Comment by Gkrumbach07** [comment]
changes look good. PR only applies UX approved changes here https://github.com/opendatahub-io/odh-dashboard/pull/2518#discussion_r1505431626. The other convo in jira 3887 is still under review.

/lgtm

**Pull request merged**